.github/workflows/sync-sendgripd-template.yaml

```diff
@@ -29,4 +32,4 @@ jobs:
         uses: actions-js/push@master
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
-          branch: jyg-UID2-833-setup-sendgrid
+          branch: jyg-UID2-833-TAM_email_template
```

Comment:
Does the branch have to be coded here ? Can it not be selected from the dropdown while running the workflow?

Reply:
The workflow_dispatch does not work until this is merged into main. I will remove this